src/main/java/frc/robot/RobotContainer.java

```diff
@@ -62,56 +60,63 @@ public RobotContainer() {
         // The left stick controls translation of the robot.
         // Turning is controlled by the X axis of the right stick.
         new RunCommand(
```

Comment:
You could make this `m_robotDrive.run(()->…)` to line up with wpilib recommendations and handle the requirement implicitly.